.github/DISCUSSION_TEMPLATE/new.yml

```diff
@@ -0,0 +1,61 @@
+name: New Prisma Discussion
+description: Please fill out the following information to create a new discussion related to Prisma.
+  Make sure to provide clear and concise details to help others understand and address your concerns effectively.
+
+body:
```

Comment:
I kind of like this intro in the bug reports template, maybe we should have smth similar?

<img width="682" alt="image" src="https://github.com/prisma/prisma/assets/4058327/b0e220ec-eb09-4d2d-a349-07c8684ea3cc">


```suggestion
body:
  - type: markdown
    attributes:
      value: |
        Thanks for raising the questions you have about Prisma, we are happy to help! 🙏 Please follow the sections in the template and provide as much information as possible about your problem, e.g. by [setting the `DEBUG="*"` environment variable](https://www.prisma.io/docs/concepts/components/prisma-client/debugging) and enabling [additional logging output](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/logging) in Prisma Client.   
```